pyproject.toml

```diff
@@ -1,17 +1,14 @@
 [build-system]
-requires = [
-    "setuptools >= 61",
-    'tomli; python_version < "3.11"',
-]
-build-backend = "setuptools.build_meta"
+requires = ["uv_build>=0.9.6,<0.10.0"]
+build-backend = "uv_build"
 
 [project]
 name = "pyemu"
-dynamic = ["version"]
+version = "1.4.0.dev3"
```

Comment:
`uv version --bump <BUMP>` is a handy hook for keeping this current, but releases won't be triggered until a git tag is pushed to pypest/pyemu (and the tag starts with 'v')